requirements_browser.txt

```diff
@@ -0,0 +1,2 @@
+mne-qt-browser
+PySide6
```

Comment:
I don't think this should be added here but rather `qtpy`.

If we had to pick one PyQt6 would be better anyway until PySide6 fixes the matplotlib-interactive-doesn't-work bug